frontend/react/style-guide.md

```diff
@@ -288,7 +284,10 @@ src
     });
 
     // good
-    class ReservationCard extends Component {
+    function ReservationCard() {
+      return (
+        // Some JSX
+      )
```

Comment:
maybe it will be better to do something like:

// good
`class ReservationCard....`

// better
function ReservationCard...